libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/MapboxTripNotification.kt

```diff
@@ -97,7 +97,6 @@ class MapboxTripNotification constructor(
 
         pendingOpenIntent = createPendingOpenIntent(applicationContext)
         pendingCloseIntent = createPendingCloseIntent(applicationContext)
-        buildRemoteViews()
```

Comment:
maybe still need this line..

Reply:
I think yes, it does the initialization.

Reply:
going to keep it